listenbrainz/webserver/views/recommendations_cf_recording.py

```diff
@@ -0,0 +1,190 @@
+import ujson
+import requests
+from datetime import datetime
+
+from flask import Blueprint, render_template, current_app
+from flask_login import current_user, login_required
+from listenbrainz.domain import spotify
+
+from listenbrainz.webserver.views.user import _get_user
+import listenbrainz.db.recommendations_cf_recording as db_recommendations_cf_recording
+from werkzeug.exceptions import BadRequest, InternalServerError
+
+
+recommendations_cf_recording_bp = Blueprint('recommendations_cf_recording', __name__)
+
+SERVER_URL = "https://labs.api.listenbrainz.org/recording-mbid-lookup/json"
```

Comment:
Ideally this should be a better name -- server for what??

Reply:
Cool